### ISSUE TITLE
Tabview accessibility issue: the focus moves in a specific order, focusing on the tabs and their contents sequentially

### ISSUE THREAD

**Issue by rinnuagna** [primefaces/primereact#5176]
### Describe the bug

I have a UI with multiple tabs and the content of each tab includes tables and buttons.
When I am navigating through the UI using a keyboard, the focus moves in a specific order, focusing on the tabs and their contents sequentially rather than jumping between tabs before focusing on their contents. I think the keyboard navigation should be in such a way that, it should first access a tab then is contents and then to the subsequent tab.

The focus order should be like as mentioned above according to https://www.w3.org/WAI/WCAG21/Understanding/focus-order

### PrimeReact version

10.x, 7.2.1

### React version

17.x

### Language

TypeScript

### Build / Runtime

Create React App (CRA)

**Comment by melloware** [comment]
All accessibility issues will be addressed in a future version maybe 11.0.0.  PrimeVue currently has a lot of the accessibility issues implemented and PrimeReact will be receiving those fixes.

**Comment by habubey** [comment]
Issue tracker is used for defects only as part of our commitment to quality and continuous improvement in all areas. Enhancements are collected as valuable community feedback and managed internally so moving this enhancement ticket to our internal project management backlog.

**Comment by rinnuagna** [comment]
Where can I track this enhancement?
